### PR TITLE
RATEST-90 :failing Test: Add Patient Appointment Test

### PR DESCRIPTION
**Pull request by insookwa**
https://issues.openmrs.org/browse/RATEST-90

Test Passes now
please kindly review 
 cc @k-joseph  @kdaud , @sherrif10

### CODE REVIEW

**Review by sherrif10 — APPROVED** [review]
Well done @insookwa  and wel be back again
LGTM after testing it locally cc @k-joseph

**Comment by kdaud** [comment]
> LGTM after testing it locally

@sherrif10 are you sure the changes passes with CI ?

Am skeptical about what `CI report` will produce concerning the changes !

**Comment by sherrif10** [comment]
Like i commented above this test pass locally , we can only track other results via github actions  but since he is the first committer, he needs to be approved then github actions can get activated on this pull request

**Comment by kdaud** [comment]
> this test pass locally

IMO Its a good idea `PR approvals` keep on hold until we can clearly see the CI report `(green)` on a test. 

> he needs to be approved then github actions can get activated on this pull request

Have raised this to the community [here](https://talk.openmrs.org/t/ci-build-plan-on-openmrs-modules-denies-first-time-contributors/33405/2)

File: pom.xml
Comment (sherrif10) [inline pom.xml:55]: Thanks @insookwa Please remove un wanted commits

File: pom.xml
Comment (sherrif10) [inline pom.xml:43]: To revert last commit, you can say **git log** in your terminal , the copy the first commit id which is the last on github
then say  **git revert <last commit id>.** after push

**Comment by insookwa** [comment]
fixing as recommended @sherrif10

**Pull request closed without merging**
